code/modules/games/cards/lotto.dm

```diff
@@ -0,0 +1,131 @@
+/obj/item/toy/lotto_ticket
+	name = "scratch-off lotto ticket"
+	desc = "A scratch-off lotto ticket."
+	icon = 'icons/obj/toy.dmi'
+	w_class = W_CLASS_TINY
+	var/revealed = FALSE
+	var/iswinner = FALSE
+	var/ticket_price
+	var/winnings = 0
+	var/flashed = FALSE
```

Comment:
This seems to only be used by the `supermatter_surprise` subtype, so it should be moved there.

Reply:
moved to the subtype